<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookManager</class>
 <widget class="QWidget" name="BookManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>961</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="BookInformationManagerTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>  Âõæ‰π¶‰ø°ÊÅØÁÆ°ÁêÜ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchpushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Êü•ËØ¢üîç</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SurepushButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>650</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Á°ÆËÆ§‚úî</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddpushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ê∑ªÂä†‚ûï</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>701</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="BookID">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Âõæ‰π¶ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="BookIDlineEdit"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="BookName">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Âõæ‰π¶ÂêçÂ≠ó</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="BookNamelineEdit"/>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="WriterName">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>‰ΩúËÄÖÂêçÂ≠ó</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="WriterNamelineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="DeletepushButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>9</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>√ó</string>
   </property>
  </widget>
  <widget class="QTableView" name="BookInformationtableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>961</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AllpushButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ÂÖ®ÈÉ®</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
